dotcom-rendering/src/web/components/ShadyPie.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { ShadyPie } from './ShadyPie';
+
+const defaultFormat: ArticleFormat = {
+	theme: ArticlePillar.News,
+	design: ArticleDesign.Standard,
+	display: ArticleDisplay.Standard,
+};
+
+const labsFormat: ArticleFormat = {
+	theme: ArticlePillar.Lifestyle,
+	design: ArticleDesign.Standard,
+	display: ArticleDisplay.Standard,
+};
+
+export default {
+	component: ShadyPie,
+	title: 'Components/ShadyPie',
+};
+
+export const Default = () => {
+	return (
+		<div style={{ width: '300px' }}>
+			<ShadyPie format={defaultFormat} editionId={'UK'} />
+		</div>
+	);
+};
+Default.story = {
+	name: 'Default',
+};
```

Comment:
Just a thought, what happens if you don't supply either `format` or `editionId`? Given these are optional is that the true default?

Reply:
It is the true default - turns out we don't need to supply format or editionId, so I've simplified the code for this story now